Documentation/Functions/Data.rst

```diff
@@ -1000,7 +1000,15 @@ siteLanguage
         The language mapped to the ID of the site language.
 
     :typoscript:`locale`
-        The locale, like `de_CH` or `en_GB`.
+        ..  versionchanged:: 12.3
+            Several subkeys are available:
+
+            *  :typoscript:`languageCode`: this contains the two-letter language code (previously :typoscript:`siteLanguage:twoLetterIsoCode`)
```

Comment:
I have tried following:
```page.30 = TEXT
page.30.data = SITELANGUAGE:locale
page.35 = TEXT
page.35.value = <br>

page.40 = TEXT
page.40.data = SITELANGUAGE:locale.languageCode
page.45 = TEXT
page.45.value = <br>

page.50 = TEXT
page.50.data = SITELANGUAGE:locale.countryCode
page.55 = TEXT
page.55.value = <br>

page.60 = TEXT
page.60.data = SITELANGUAGE:locale.full
page.65 = TEXT
page.65.value = <br>
```

It results in:
```
en_US.UTF-8
en-US
en-US
en-US
```

No case results in just "en", the mentioned twoLetterISOcode.

Reply:
So is this a Core bug or an documentation issue? I just preported this PR for someone else...

Reply:
We will take care of the problem in a follow up